src/main/java/ch/wisv/events/core/model/order/OrderProductDto.java

```diff
@@ -10,11 +10,13 @@ public class OrderProductDto {
      * Field product should contain Product keys and number of webshop of that product.
      */
     HashMap<String, Long> products;
+    Boolean agreedGTC;
```

Comment:
Is it necessary to use `Boolean` here?

Reply:
Instead of using bool you mean? I always learned to only use Object types instead of native types in models in spring, because native types often cause trouble with the database.